docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -174,13 +174,13 @@ make deploy/service IMAGE_TAG=<your-image-tag-here> <OPTIONAL_PARAMETERS>
 - `OBSERVATORIUM_RHSSO_TENANT`: Tenant of an Observatorium using RHSSO as authentication. Defaults to `''`.
```

Comment:
Since the `OBSERVATORIUM_RHSSO_GATEWAY` is going to be removed, it can also be removed from this list.